### PR TITLE
CTPPS: AOD EventContent update

### PR DESCRIPTION
**Pull request by jan-kaspar**
- `RecoCTPPS/Configuration/python/RecoCTPPS_EventContent_cff.py`: AOD has the same EventContent as RECO. The added products are negligible in size and could be important for commissioning.
- `RecoCTPPS/Configuration/test/raw_data_test.py`: a test config for validating the standard cff files.

This PR should be merged after #14605.

### CODE REVIEW

File: RecoCTPPS/Configuration/python/recoCTPPS_cff.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
```
Comment (slava77) [inline RecoCTPPS/Configuration/python/recoCTPPS_cff.py]: this file already exists in #14605

**Comment by slava77** [comment]
please put these changes in #14605

**Comment by slava77** [comment]
why do we need to split integral parts of #14605 into a separate PR?
(this will then also mean two PRs to backport to 80X and do it in a specific order)

**Comment by jan-kaspar** [comment]
I forgot that `RecoCTPPS/Configuration/python/` were already in #14605. I will move this PR there.

**Comment by slava77** [comment]
On 6/9/16 7:32 AM, jan-kaspar wrote:
> I forgot that |RecoCTPPS/Configuration/python/| were already in #14605
> <https://github.com/cms-sw/cmssw/pull/14605>. I will move this PR there.
> 

Thank you.

... almost there...

**Comment by jan-kaspar** [comment]
Changes proposed in this PR have been moved to #14605.

**Pull request closed without merging**